src/fragments/lib/graphqlapi/js/working-with-files.mdx

```diff
@@ -0,0 +1,910 @@
+The Storage and GraphQL API categories can be used together to associate a file, such as an image or video, with a particular record. For example, you might create a `User` model with a profile picture, or a `Post` model with an associated image. With Amplify's GraphQL API and Storage categories, you can reference the file within the model itself to create an association.
+
+To get started, run the following in an existing Amplify project:
+
+```bash
+npm i @aws-amplify/ui-react  #For Authenticator component
+amplify add auth             #Select default configuration
+amplify add storage          #Select S3 with read/write access
+amplify add api              #Select Cognito User Pool for authorization type
+```
+
+When prompted, use the following schema:
+
+```graphql
+type Song @model @auth(rules: [{ allow: public}]) {
+  id: ID!
+  name: String!
+  covertArtKey: String #Set as optional to allow adding file after initial create
```

Comment:
Should we clarify here the "how" something is set as optional? Something like: 
"Set as optional by leaving out the trailing '!' to allow adding file after initial create"

Reply:
Because this isn't a standalone tutorial and is in the context of our other docs, I try to refrain from going into too much unrelated detail that would clutter the examples / explanations. If the customer needs more details on creating / modifying a schema, my assumption is that they will visit the relevant docs pages. Since there is a required `name` field above this, it also seems fairly straightforward by doing a quick visual comparison.